Extension/package.json

```diff
@@ -258,8 +257,7 @@
         "source": "gcc",
         "owner": "cpptools",
         "fileLocation": [
-          "autoDetect",
-          "${workspaceFolder}"
+          "autoDetect"
```

Comment:
https://github.com/microsoft/vscode/blob/28ac2de7d0f2b5298957c0c42c3b06230e885726/src/vs/platform/markers/common/problemMatcher.ts -- I think maybe replace ${cwd} since that's what VS Code seems to be doing.